src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -3742,10 +3741,10 @@ namespace LowTempRadiantSystem {
         Real64 ZeroFlowSurfTemp;  // Temperature of radiant surface when flow is zero
         int ZoneNum;              // Zone pointer for this radiant system
 
-        VarFlowRadDesignData variableFlowDesignDataObject{
```

Comment:
The only thing more annoying than unnecessary constructor notation, is unnecessary list constructor notation.